flathunter/detailed_hunter.py

```diff
@@ -0,0 +1,36 @@
+"""Flathunter implementation for detailed exposé with OpenAI text generation."""
+from flathunter.config import YamlConfig
+from flathunter.logging import logger
+from flathunter.hunter import Hunter
+from flathunter.filter import Filter
+from flathunter.processor import ProcessorChain
+from flathunter.exceptions import BotBlockedException, UserDeactivatedException
+
+class OpenAIHunter(Hunter):
```

Comment:
This looks fine but per my other comment I think I would prefer just to change the default Hunter.